miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/MiniAppCustomPermissionCache.kt

```diff
@@ -111,11 +111,12 @@ internal class MiniAppCustomPermissionCache constructor(
 
     @VisibleForTesting
     fun applyStoringPermissions(miniAppCustomPermission: MiniAppCustomPermission) {
-        val jsonToStore: String = Gson().toJson(sortedByDefault(miniAppCustomPermission))
+        val jsonToStore: String = Gson().toJson(miniAppCustomPermission)
         prefs.edit().putString(miniAppCustomPermission.miniAppId, jsonToStore).apply()
     }
 
     // Sort the `pairValues` by ordinal of [MiniAppCustomPermissionType].
+    // [NOT_USED]
```

Comment:
Can we remove this method if its not used ?

Reply:
@rleojoseph yes we can.